msal_extensions/cache_lock.py

```diff
@@ -39,7 +39,10 @@ def try_to_create_lock_file(self):
                     return True
             except OSError as err:
                 if err.errno == errno.EEXIST:
+                    logging.warning("Lock file exists, trying again after some time")
```

Comment:
Long story short, please change all `logging` into `logger`, after we follow [this pattern](https://github.com/AzureAD/microsoft-authentication-extensions-for-python/blob/0.3.0/msal_extensions/libsecret.py#L16-L18).